### ISSUE TITLE
strusUtilities and strusRpcServer cannot run on the storages managed by the webservice

### ISSUE THREAD

**Issue by andreasbaumann** [Eurospider/strusWebService#44]
We could base on the objects in strusModule (currently we use strus objects directly). But then
we must run an RPC server along to the strusWebService.

Alternative solution is to add JSON support to a new set and/or the existing CLI tools in
strusUtilities.

**Comment by patrickfrey** [comment]
Why do you think you need to run an RPC server, if you base on the objects in strusModule ?

**Comment by andreasbaumann** [comment]
How can I access the same index from both the strusWebService and a CLI like strusInspect?

**Comment by patrickfrey** [comment]
You are right. Running an RPC server is currently the only feasible way to access a storage from both the WebService and from CLI tools at the same time.
